inventory_service/src/update_item_container.py

```diff
@@ -6,3 +6,5 @@ def update_item_container(cursor,id,new_container):
     except Exception as e:
         return e
 
+        
+
```

Comment:
The two functions can be combined into one function. Based on the json input, the specific field can be updated as there is only one table present in the database.
 `update_item_container`
 `update_item`